apps/prairielearn/src/ee/pages/instructorAiGenerateQuestion/instructorAiGenerateQuestion.html.ts

```diff
@@ -135,3 +135,14 @@ ${generatedPython}
     </div>
   `.toString();
 };
+
+export const GenerationFailure = (urlPrefix: string, jobSequenceId: string): string => {
```

Comment:
```suggestion
export function GenerationFailure(urlPrefix: string, jobSequenceId: string): string {
```

Let's also use an object with named properties, just like we do everywhere else.

Reply:
done in 4c35422

Reply:
done in e3bef33